pkgs/development/python-modules/torchaudio/bin.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchurl
+, isPy37
+, isPy38
+, isPy39
+, patchelf
+, python
+, pytorch-bin
+}:
+
+let
+  pyVerNoDot = builtins.replaceStrings [ "." ] [ "" ] python.pythonVersion;
+  srcs = import ./binary-hashes.nix version;
+  unsupported = throw "Unsupported system";
+  version = "0.10.0";
+in buildPythonPackage {
+  inherit version;
+
+  pname = "torchaudio";
+
+  format = "wheel";
```

Comment:
It is to avoid the long build time of PyTorch CUDA builds.

Reply:
In addition to the build time, these 4 packages(torch,vision,audio,libtorch) are released by https://pytorch.org/ at the same time. I think it's better to use these together at runtime.

Reply:
> It is to avoid the long build time of PyTorch CUDA builds.

because they are unfree and not on hydra?

> In addition to the build time, these 4 packages(torch,vision,audio,libtorch) are released by pytorch.org at the same time. I think it's better to use these together at runtime.

If they are releases as binaries the sources are also release, right?

Reply:
I think they are free, but not on hydra, so build-time is long.
https://github.com/pytorch/pytorch/blob/master/LICENSE

> If they are releases as binaries the sources are also release, right?

Right.

Reply:
The main reason to use binary is that it is not in hydra.
There's nothing we can't build from source, but it can take up to 6 hours to build, and we'll need more time to try with cpu and cuda.
There is a huge loss of time when adding new features and changing build option every 3 months.

Reply:
--
Change the license to bsd.